homeassistant/components/izone/climate.py

```diff
@@ -569,3 +607,15 @@ async def async_turn_off(self):
         """Turn device off (close zone)."""
         await self._controller.wrap_and_catch(self._zone.set_mode(Zone.Mode.CLOSE))
         self.async_write_ha_state()
+
+    @property
+    def zone_index(self):
```

Comment:
We don't need this property. Just use the instance attribute directly where needed.

Reply:
Removed